src/lib/error.ts

```diff
@@ -6,6 +6,11 @@ export enum ErrorCode {
     AuthenticationRequired = 'You are not authenticated!',
     UnsupportedAuthorization = 'Unsupported authorization type!',
     MissingRefreshToken = 'Refresh token is required parameter! Please send it in body or query as refreshToken.',
+    UserByResetTokenNotImplemented = "Function 'getUserByResetPasswordToken' is not implemented!",
```

Comment:
Change this "not implemented" to something more clear, that the user haven't passed in the required settings for the feature.

Reply:
Sorry this is old, I forgot to remove it. This "functionality" could be supplied by the `validateResetToken` function. Because the save token function is optional - we dont need to have this function. If you want to get the token & user from database and validate it you can do it inside the `validateResetToken` function.